File: 배열/프로그래머스lv1-k번째 수.py
def solution(array, commands):
    result = []
    for i, j, k in commands:
        result.append(sorted(array[i - 1:j])[k - 1])
    
    return result

"""
정확성  테스트
테스트 1 〉	통과 (0.00ms, 10.1MB)
테스트 2 〉	통과 (0.01ms, 10.1MB)
테스트 3 〉	통과 (0.01ms, 10.3MB)
테스트 4 〉	통과 (0.00ms, 10.1MB)
테스트 5 〉	통과 (0.01ms, 10.1MB)
테스트 6 〉	통과 (0.00ms, 10.2MB)
테스트 7 〉	통과 (0.01ms, 9.9MB)
"""

File: 배열/프로그래머스lv1-k번째 수.js
function solution(array, commands) {
  return commands.map(([i, j, k]) => {
    const arr = array.slice(i - 1, j).sort((a, b) => a - b)
    return arr[k - 1]
  })
}
// 정확성 테스트
// 테스트 1 〉 통과 (0.07ms, 33.5MB)
// 테스트 2 〉 통과 (0.08ms, 33.5MB)
// 테스트 3 〉 통과 (0.12ms, 33.6MB)
// 테스트 4 〉 통과 (0.11ms, 33.6MB)
// 테스트 5 〉 통과 (0.08ms, 33.4MB)
// 테스트 6 〉 통과 (0.07ms, 33.5MB)
// 테스트 7 〉 통과 (0.12ms, 33.5MB)
